docs/preview/new-system.md

```diff
@@ -30,6 +30,12 @@ With Docker running, follow these steps to preview your documentation site local
     docker-compose up -d
     ```
 
+    If you don't already have a `.env` file this will throw an error and you'll need to create one:
+
+    ```bash
+    touch .env
+    ```
```

Comment:
and then run the docker-compose command again

Reply:
Correct. The contributor receives this error when running docker-compose...
`ERROR: Couldn't find env file: /path/to/project/.env`

...so I figured it was pretty obvious you need to rerun it after creating the file. What I thought might not be so obvious is how to create the file or what the contents should be. I have pushed a commit to clarify